server/services/store/sqlstore/migrations/migrations_files/000001_init.up.sql

```diff
@@ -12,4 +12,4 @@ CREATE TABLE IF NOT EXISTS {{.prefix}}blocks (
 	update_at BIGINT,
 	delete_at BIGINT,
 	PRIMARY KEY (id, insert_at)
-){{if .mysql}}CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci{{end}};
```

Comment:
Did we come to a conclusion on what (if any) encoding to set? Per @jespino's last comment in [this thread](https://community.mattermost.com/core/pl/tyr59rq8wb8sifxqgka8gybeqr): "Yes, the problem is, if you not "hardcoded" it, it can use utf8 and everything looks broken, we decided to choose one collation there, the user can still alter the table later. I'm not saying that decision is correct, but not setting it can lead to another kind of problems"

Reply:
@chenilim both charset and collation can get set in DB connection strings in Focalboard config files. So one doesn't need to hardcode them in here. 

One thing we need to do is document it in Focalboard docs to use `utf8mb4_general_ci` collation or any other `utf8mb4` variant. Can you point me to docs where I can document this?

Reply:
I'm not 100% comfortable with this change, but I would need to think more about it.

Reply:
@jespino whats your concern?

Reply:
Resolving as we're fine with this approach and adding a default collation.